includes/database/engine/class-query.php

```diff
@@ -1451,10 +1507,10 @@ protected function shape_items( $items = array() ) {
 		 *
 		 * Looks like `edd_get_customers`
 		 *
-		 * @since 3.0
+		 * @since 1.0.0
 		 *
 		 * @param array  $retval An array of items.
-		 * @param Query &$this  Current instance of Query, passed by reference.
+		 * @param object &$this  Current instance of Query, passed by reference.
```

Comment:
```suggestion
		 * @param Query &$this  Current instance of Query, passed by reference.
```

I prefer to use explicit class names instead of generic `object`.